Assets/LeapMotion/Resources/VertexOffsetShader.shader

```diff
@@ -0,0 +1,49 @@
+﻿Shader "LeapMotion/VertexOffsetShader" {
+	Properties{
+		_Color("Color", Color) = (1, 1, 1, 1)
+		_MainTex("Albedo (RGB)", 2D) = "white" {}
+		_Glossiness("Smoothness", Range(0, 1)) = 0.5
+		_Metallic("Metallic", Range(0, 1)) = 0.0
+		[MaterialToggle] _isLeftHand("Is Left Hand?", Int) = 0
+	}
+	SubShader{
+			Tags{ "RenderType" = "Opaque" }
+			LOD 200
+
+			CGPROGRAM
+			// Physically based Standard lighting model, and enable shadows on all light types
+#pragma surface surf Standard fullforwardshadows vertex:vert
+
+			// Use shader model 3.0 target, to get nicer looking lighting
+#pragma target 3.0
+
+			uniform float4x4 _handTransforms[2];
```

Comment:
Since this constant is global, we should make extra sure that there are no collisions.  I also think it would be good to follow the same naming convention used by LeapCG, by prefixing everything by _Leap and such.